StreamChatClient_v3/APIClient/Endpoints/Requests/GuestUser.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// Type that serves guest-user params
+struct GuestUser<ExtraData: UserExtraData>: Encodable {
+    private enum CodingKeys: String, CodingKey {
+        case id
+    }
+
+    /// User's identifier
+    let id: UserId
+    /// User's extra data
+    let extraData: ExtraData?
```

Comment:
`ExtraData` is not an optional field in v3

Reply:
fixed in 0f7845d545017e8c34d738d2123f49509eb45efe